src/rest-server/src/models/v2/virtual-cluster/index.js

```diff
@@ -20,5 +20,10 @@ const launcherConfig = require('@pai/config/launcher');
 if (launcherConfig.type === 'yarn') {
   module.exports = require('@pai/models/v2/virtual-cluster/yarn');
 } else {
-  module.exports = require('@pai/models/v2/virtual-cluster/yarn');
+  const yarnModel = require('@pai/models/v2/virtual-cluster/yarn');
```

Comment:
remove

Reply:
It's a workaround to avoid unnecessary copy, and we could overwrite yarn's implementation one by one